tools/syz-covermerger/syz_covermerger.go

```diff
@@ -22,14 +22,15 @@ import (
 var (
 	flagWorkdir = flag.String("workdir", "workdir-cover-aggregation",
 		"[optional] used to clone repos")
-	flagRepo      = flag.String("repo", "", "[required] repo to be used as an aggregation point")
-	flagBranch    = flag.String("branch", "", "[required] branch to be used as an aggregation point")
-	flagCommit    = flag.String("commit", "", "[required] commit hash to be used as an aggregation point")
-	flagNamespace = flag.String("namespace", "upstream", "[optional] target namespace")
-	flagDuration  = flag.Int64("duration", 0, "[optional] used to mark DB records")
-	flagDateTo    = flag.String("date-to", "", "[optional] used to mark DB records")
-	flagTotalRows = flag.Int64("total-rows", 0, "[optional] source size, is used for version contol")
-	flagToDashAPI = flag.String("to-dashapi", "", "[optional] dashapi address")
+	flagRepo                = flag.String("repo", "", "[required] repo to be used as an aggregation point")
+	flagBranch              = flag.String("branch", "", "[required] branch to be used as an aggregation point")
+	flagCommit              = flag.String("commit", "", "[required] commit hash to be used as an aggregation point")
+	flagNamespace           = flag.String("namespace", "upstream", "[optional] target namespace")
+	flagDuration            = flag.Int64("duration", 0, "[optional] used to mark DB records")
+	flagDateTo              = flag.String("date-to", "", "[optional] used to mark DB records")
+	flagTotalRows           = flag.Int64("total-rows", 0, "[optional] source size, is used for version contol")
+	flagToDashAPI           = flag.String("to-dashapi", "", "[optional] dashapi address")
+	flagDashboardClientName = flag.String("-dashboard-client-name", "coverage-merger", "[optional]")
```

Comment:
Please remove the leading dash from the flag name.

Reply:
done, thanks